src/app/features/tasks/add-task-bar/add-task-bar.component.ts

```diff
@@ -149,6 +149,17 @@ export class AddTaskBarComponent implements AfterViewInit, OnDestroy {
         }
       });
     });
+
+    const savedTodo = this._taskService.getSavedTodo();
+    if (savedTodo)
+    {
+      this._taskService.setSavedTodo('');
+      this.taskSuggestionsCtrl.setValue(savedTodo);
+      setTimeout(() => {
```

Comment:
We probably want to store this away and clear it in `ngOnDestroy` like the other timeouts like `this._blurTimeout`. This is needed to avoid this code is being executed, after the component was destroyed, which would lead to an error.